### PR TITLE
[SPARK-43624][PS][CONNECT] Add `EWM` to SparkConnectPlanner.

### PR DESCRIPTION
**Pull request by itholic**
### What changes were proposed in this pull request?

This PR proposes to add `EWM` for SparkConnectPlanner.

### Why are the changes needed?

To increase pandas API coverage

### Does this PR introduce _any_ user-facing change?

No, we added `EWM` to SparkConnectPlanner, but there is still unresolved `AnalysisException` issues(SPARK-43611) that need to be addressed in follow-up work.

### How was this patch tested?

Manually checked the plan was created with EWM properly.

### CODE REVIEW

File: python/pyspark/pandas/tests/connect/test_parity_ewm.py [hunk 1]
```diff
@@ -22,11 +22,15 @@
 
 
 class EWMParityTests(EWMTestsMixin, PandasOnSparkTestUtils, ReusedConnectTestCase, TestUtils):
-    @unittest.skip("TODO(SPARK-43624): Enable ExponentialMovingLike.mean with Spark Connect.")
+    @unittest.skip(
+        "TODO(SPARK-43611): Fix unexpected `AnalysisException` from Spark Connect client."
+    )
```
Comment (itholic) [inline python/pyspark/pandas/tests/connect/test_parity_ewm.py:27]: Before this fix, this test was JVM-dependent, but now it is generating the same error as the unexpected `AnalysisException` posted in SPARK-43611. Will address the related tickets collectively in SPARK-43611 at once.

**Comment by itholic** [comment]
cc @zhengruifeng Could you take a look when you find some time as an original author for EWM in JVM side.

**Review by zhengruifeng — APPROVED** [review]

File: python/pyspark/pandas/window.py [hunk 2]
```diff
@@ -2448,11 +2449,26 @@ def mean(self) -> FrameLike:
         unified_alpha = self._compute_unified_alpha()
 
         def mean(scol: Column) -> Column:
-            sql_utils = SparkContext._active_spark_context._jvm.PythonSQLUtils
+            if is_remote():
+                from pyspark.sql.connect.functions import _invoke_function_over_columns, lit
+
+                col_ewm = _invoke_function_over_columns(
+                    "ewm",
```
Comment (zhengruifeng) [inline python/pyspark/pandas/window.py:2456]: nit: I guess we can name it a bit more ps-specific `pandas_ewm`

**Comment by zhengruifeng** [comment]
merged to master

File: python/pyspark/pandas/window.py (hunk 2 above)
Comment (itholic) [reply python/pyspark/pandas/window.py:2456]: Oh, I just checked the comments. Will update them as a follow-up PRs.

Comment (zhengruifeng) [reply python/pyspark/pandas/window.py:2456]: oh, no problem, on second thought, I think this naming and PR is fine enough

Comment (itholic) [reply python/pyspark/pandas/window.py:2456]: Got it. Thanks for the review! :-)

**Comment by LuciferYang** [comment]
image

An interesting character :)

**Comment by itholic** [comment]
@LuciferYang Interesting. It's not displayed from my side. 😮 

Screenshot 2023-06-21 at 1 43 14 PM

**Comment by itholic** [comment]
Oh, it's only displayed on commits list. Interesting...

**Comment by zhengruifeng** [comment]
lol, should be a bug in Github

**Pull request closed without merging**